volatility3/framework/plugins/windows/hollowprocesses.py

```diff
@@ -0,0 +1,188 @@
+# This file is Copyright 2024 Volatility Foundation and licensed under the Volatility Software License 1.0
+# which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
+#
+import logging
+from typing import NamedTuple
+
+from volatility3.framework import interfaces, exceptions, constants
+from volatility3.framework import renderers
+from volatility3.framework.configuration import requirements
+from volatility3.framework.objects import utility
+from volatility3.plugins.windows import pslist, vadinfo
+
+vollog = logging.getLogger(__name__)
+
+VadInfo = NamedTuple(
+    "VadInfo",
+    [
+        ("protection", str),
+        ("path", str),
+    ],
+)
+
+DLLInfo = NamedTuple(
+    "DLLInfo",
+    [
+        ("path", str),
+    ],
+)
+
+
+class HollowProcesses(interfaces.plugins.PluginInterface):
+    """Lists hollowed processes"""
+
+    _required_framework_version = (2, 4, 0)
+
+    @classmethod
+    def get_requirements(cls):
+        # Since we're calling the plugin, make sure we have the plugin's requirements
+        return [
+            requirements.ModuleRequirement(
+                name="kernel",
+                description="Windows kernel",
+                architectures=["Intel32", "Intel64"],
+            ),
+            requirements.ListRequirement(
+                name="pid",
+                element_type=int,
+                description="Process IDs to include (all other processes are excluded)",
+                optional=True,
+            ),
+            requirements.VersionRequirement(
+                name="pslist", component=pslist.PsList, version=(2, 0, 0)
+            ),
+            requirements.VersionRequirement(
+                name="vadinfo", component=vadinfo.VadInfo, version=(2, 0, 0)
+            ),
+        ]
+
+    def _get_vads_map(self, proc):
```

Comment:
Is this significantly different than `get_vad_map` staticmethod in `windows.vadyarascan`?  If so, it might be worth renaming this one to something a bit more specific (the last one is public and part of the API so would need version bumps to change).

Reply:
I renamed it since it gets a different set of information.